src/test/java/com/spotify/confidence/FeatureProviderTest.java

```diff
@@ -96,19 +97,16 @@ static void after() {
 
   @Test
   public void missingTargetKey() {
-
     final FlagEvaluationDetails<Value> evaluationDetails =
         client.getObjectDetails("flags/whatever", DEFAULT_VALUE);
 
-    assertThat(evaluationDetails.getValue()).isEqualTo(DEFAULT_VALUE);
-    assertThat(evaluationDetails.getErrorCode()).isEqualTo(ErrorCode.TARGETING_KEY_MISSING);
+    assertThat(evaluationDetails.getErrorCode()).isEqualTo(GENERAL);
     assertThat(evaluationDetails.getReason()).isEqualTo("ERROR");
     assertThat(evaluationDetails.getVariant()).isBlank();
   }
 
   @Test
   public void inconsistentTargetKey() {
```

Comment:
When running just this test in the editor it fails.

Reply:
is it the same on master?